src/app/containers/ConsentBanner/index.amp.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import styled from 'styled-components';
+import Banner from './Banner/index.amp';
+
+const parentId = 'consent';
+const promptId = 'consent-prompt';
+const privacyId = 'privacy';
+const cookieId = 'cookie';
+
+const ampGeoData = {
+  ISOCountryGroups: {
+    eu: [
+      'at',
+      'be',
+      'bg',
+      'cy',
+      'cz',
+      'de',
+      'dk',
+      'ee',
+      'es',
+      'fi',
+      'fr',
+      'gb',
+      'gr',
+      'hr',
+      'hu',
+      'ie',
+      'is',
+      'it',
+      'li',
+      'lt',
+      'lu',
+      'lv',
+      'mt',
+      'nl',
+      'no',
+      'pl',
+      'pt',
+      'ro',
+      'se',
+      'si',
+      'sl',
+    ],
+  },
+};
+
+const ampConsentData = {
+  consents: {
+    'user-consent': {
+      promptIfUnknownForGeoGroup: 'eu',
+      promptUI: promptId,
+    },
+  },
+};
+
+const jsonInlinedScript = data => (
+  <script
+    type="application/json"
+    /* eslint-disable-next-line react/no-danger */
+    dangerouslySetInnerHTML={{ __html: JSON.stringify(data) }}
+  />
+);
+
+// Style `amp-consent` as child due to inability to set
+// `layout` attribute on styled `amp-consent` component
+const AmpConsentWrapper = styled.div`
+  & amp-consent {
+    position: initial;
+    display: block;
+  }
+`;
+
+const Amp = () => (
+  <AmpConsentWrapper>
+    <amp-geo layout="nodisplay">{jsonInlinedScript(ampGeoData)}</amp-geo>
+    <amp-consent id={parentId} layout="nodisplay">
+      {jsonInlinedScript(ampConsentData)}
+      <div id={promptId}>
+        <Banner
+          type="privacy"
+          acceptAction={{ tap: [`${cookieId}.show, ${privacyId}.hide`] }}
+          rejectAction={{ tap: [`${cookieId}.show, ${privacyId}.hide`] }}
+          promptId={privacyId}
```

Comment:
Shouldn't this have a `hidden` attribute too?

```jsx
hidden={hasUserBeenHereBefore() ? null : true}
```

Reply:
Nope, amp js does the hiding for us, hidden just tells amp what the expected visual is